app.js

```diff
@@ -547,3 +547,17 @@ if (desktopCards) {
     });
   });
 }
+
+// contact form validation
+
+const form = document.querySelector('.contact-form');
+const errorDiv = document.getElementById('error-message');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const email = document.querySelector('.input-email').value;
+  if (email === email.toLowerCase()) {
+    form.submit();
+  } else {
+    errorDiv.innerText = 'The email must be in lowcase';
+  }
+});
```

Comment:
Great work by implementing this functionality it works just as expected 🥇 
[OPTIONAL] one small thing to add is removing the error message when the user changes the input content as it is confusing to keep it while a valid email is entred this is important for good UI

![Capture d’écran du 2023-08-05 10-59-04](https://github.com/FilbertKimasa/Portfolio/assets/59707859/38cdf9a8-418a-488e-9fc9-d4f2b0f12ad1)